src/main/java/com/minecolonies/coremod/client/gui/WindowRequestDetail.java

```diff
@@ -0,0 +1,241 @@
+package com.minecolonies.coremod.client.gui;
+
+import com.minecolonies.api.colony.requestsystem.manager.IRequestManager;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.minecolonies.api.colony.requestsystem.manager.IRequestManager'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AUselessImportCheck)